### ISSUE TITLE
[Hint] Intrinsicsize

### ISSUE THREAD

**Issue by molant** [webhintio/hint#2019]
# 🚀 Hint request

## Description

The intrinsic size attribute allows developers to specify the intrinsic size of a media element (<img>, <video>, <svg:image>). With this attribute media elements will maintain their aspect ratios. Developers need only specify one dimension, either a percentage or pixel value, and the other dimension will be computed immediately without causing a visual re-flow. 

 Are you willing to work on this?

Yup

## What should be tested exactly?

TBD how this should work. Need a bit more of research.

## Relevant links

https://googlechrome.github.io/samples/intrinsic-size/
https://github.com/WICG/intrinsicsize-attribute
https://24ways.org/2018/jank-free-image-loads/
https://twitter.com/search?q=intrinsic%20size&src=typd

**Comment by Malvoz** [comment]
This issue should be closed. 

The proposal for the attribute is [no longer being pursued](https://github.com/WICG/intrinsicsize-attribute/pull/18) by WICG, as a result of the conclusion in WICG/intrinsicsize-attribute#16.

### tldr

This is instead solved by UAs adding below styles to the UA stylesheet, making the attribute obsolete:
```CSS
img, video {
    aspect-ratio: attr(width) / attr(height);
} 
```

**Comment by molant** [comment]
I saw it mentioned at one talk during View Source yesterday but forgot about updating this issue.

Thanks!
